tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,4 @@
+export function trimWhiteSpaces(str){
+    let res = str.split(" ").join('')
```

Comment:
Use const instead of let in this case

Reply:
I guess we need to use consistent quotes across the code base. Single quotes is good to see here